src/main/java/org/apache/sling/resourceresolver/impl/CommonResourceResolverFactoryImpl.java

```diff
@@ -516,6 +494,16 @@ public ResourceProviderTracker getResourceProviderTracker() {
         return authenticationInfo;
     }
 
+    @Override
```

Comment:
Can we maybe use more inclusive language here like allow/deny list?

Reply:
Updated